db/migrations/2023_12_12_b1deb87ac586_add_birthdays.py

```diff
@@ -0,0 +1,46 @@
+"""add birthdays
+
+Revision ID: b1deb87ac586
+Revises: 84abfe2ccee0
+Create Date: 2023-12-12 21:15:21.827946
+
+"""
+from typing import Sequence, Union
+
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision: str = 'b1deb87ac586'
+down_revision: Union[str, None] = '84abfe2ccee0'
+branch_labels: Union[str, Sequence[str], None] = None
+depends_on: Union[str, Sequence[str], None] = None
+
+
+def upgrade() -> None:
```

Comment:
I haven't actually tested this migration and probably should 🙃

Reply:
hahaha generally advisable

Reply:
Okay, okay, I tested it and it works 👍